docs/production-deployment/cloud/export.mdx

```diff
@@ -26,6 +26,7 @@ tags:
 :::tip Support, stability, and dependency info
 
 - Workflow History Export is in a Public Preview release status for Temporal Cloud.
+- Workflow History Export does not currently support Namespaces in Google Cloud.
```

Comment:
Is the intent to have the whole GCP documentation ready, but still add a mention it's not supported here?
Or should we have the documentation for GCP ready, but just merge the part saying it's not supported for now?

Wondering because right now it could be confusing if I get directly to the part below saying `you can configure your workflow history export in AWS and GCP currently`.

Reply:
Also wondering if the `this is not supported in GCP` should be in a warning block vs. a tip block; tip blocks are easier to miss in docusaurus

Reply:
@fairlydurable What do you recommend here?

Reply:
We do not use "currently" as that is a promise of future changes in direction. Docs only reflects items in our release stages

Reply:
Sounds good.